### DISCUSSION TITLE
Incoming message bounces for one mail account

### DISCUSSION THREAD

**Discussion by adrian-abit** [modoboa/modoboa#2097] [Help]
# Impacted versions

* OS Type: Debian
* OS Version: 10.4
* Database Type: MySQL
* Database version: 15.1
* Modoboa: 1.17.0
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
Install Modoboa, add an account and try to send an email to the account
# Current behavior
Mail bounces when sent to admin@abit.dev with the message: `550 5.1.1 <admin@abit.dev>: Recipient address rejected: undeliverable address: mail for abit.dev loops back to myself`

Mails sent from admin@abit.dev get delivered correctly.

Mails to postmaster@adrianbit.tech, test@abit.dev etc. go through. Only this account bounces. 
# Expected behavior
Mail should go through.
# Video/Screenshot link (optional)

**Comment by tonioo** [comment]
@adrian-abit Looks like your @abit.dev domain is listed more than once in postfix configuration... What are your values for myhostname and mydestination in main.cf file?

**Reply by adrian-abit** [reply]
This is the config in the main.cf file

```
# This file was automatically installed on 2020-12-09T13:49:09.316538
inet_interfaces = all
inet_protocols = ipv4
myhostname = mail.adrianbit.tech
myorigin = $myhostname
mydestination = $myhostname
mynetworks = 127.0.0.0/8
smtpd_banner = $myhostname ESMTP
biff = no
unknown_local_recipient_reject_code = 550
unverified_recipient_reject_code = 550
```

**Comment by adrian-abit** [comment]
It just started to work. Mails go thru now. Haven't changed anything.
